client/app/ip/ip/order/PCC.html

```diff
@@ -1,124 +1,104 @@
                     data-translate="global_select">
-            </option>
-        </select>
-    </div>
-
-    <div class="form-group">
-        <label class="control-label"
-               for="countrySelect"
-               data-translate="ip_order_country">
+                </option>
+            </select>
+            <span class="oui-icon oui-icon-chevron-down" aria-hidden="true"></span>
         </label>
-        <select id="countrySelect"
+    </oui-field>
+
+    <oui-field data-label="{{:: 'ip_order_country' | translate }}" data-size="xl">
+        <label class="oui-select">
+            <select
+                id="countrySelect"
                 name="countrySelect"
-                class="form-control"
                 data-ng-model="model.params.country"
-                data-ng-options="c as ('country_' + c.toUpperCase() | translate) for c in orderableIp.countries"
-                data-ng-required="true">
-            <option value=""
+                class="oui-select__input"
+                data-ng-options="c as ('country_' + c.toUpperCase() | translate) for c in orderableIp.countries"                
+                data-required>
+                <option
+                    data-value=""
                     data-ng-disabled="true"
                     data-translate="global_select">
-            </option>
-        </select>
-    </div>
-
-    <div class="form-group"
-         data-ng-class="{
-             'has-error': orderIpPCC.networkName.$dirty && orderIpPCC.networkName.$invalid
-         }">
-        <label class="control-label"
-               for="networkName"
-               data-translate="ip_order_networkname">
+                </option>
+            </select>
+            <span class="oui-icon oui-icon-chevron-down" aria-hidden="true"></span>
         </label>
-        <input type="text"
-               class="form-control"
-               id="networkName"
-               name="networkName"
-               data-ng-model="model.params.networkName"
-               maxlength="20"
-               data-ng-pattern="/^[a-zA-Z]+\w+$/" />
-        <span class="fa fa-info-circle"
-           data-tooltip-placement="left"
-           data-uib-tooltip="{{ 'ip_order_networkname_help' | translate }}">
-        </span>
+    </oui-field>
+
+    <div class="oui-field">
+        <label for="control-label" class="oui-label" data-translate="ip_order_networkname"></label>
+        <div class="oui-field__content">
+            <div class="oui-field-control oui-field-control_5 d-flex">
+                <input type="text" class="oui-input"
+                       data-ng-pattern="/^[a-zA-Z]+\w+$/"
+                       data-ng-model="model.params.networkName"
+                       name="networkName"
+                       id="networkName"
+                       data-maxlength="20">
+                <span class="fa fa-info-circle m-2" data-oui-tooltip="{{:: 'ip_order_networkname_help' | translate }}"></span>
```

Comment:
```suggestion
                <span class="oui-icon oui-icon-info_circle oui-icon_bicolor m-2" data-oui-tooltip="{{:: 'ip_order_networkname_help' | translate }}"></span>
```
? 
Also would it make sense to replace this `tooltip` by a `oui-field` with a  `label-popover` ?

Reply:
A span is not `focusable`, you should not set a `tooltip` on it. You can use a `oui-popover` instead